.gitignore

```diff
@@ -8,6 +8,9 @@
 *.iws
 *.ipr
 
+*gma-create-all.sql
```

Comment:
why are these important?

Reply:
These are auto-generated by Ebean test framework, thus we want to omit them: https://ebean.io/docs/ddl-generation/